Sample_v3/Samples/CombineSimpleChat/CombineSimpleChannelsViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Combine
+import StreamChatClient
+import UIKit
+
+class CombineSimpleChannelsViewController: UITableViewController {
+    private lazy var longPressRecognizer = UILongPressGestureRecognizer(
+        target: self,
+        action: #selector(handleLongPress)
+    )
+
+    lazy var channelListController: ChannelListController = chatClient
+        .channelListController(query: ChannelListQuery(
+            filter: .in("members", [chatClient.currentUserId]),
+            pagination: [.limit(25)],
+            options: [.watch]
+        ))
+    
+    var detailViewController: CombineSimpleChatViewController?
+    
+    @available(iOS 13, *)
+    private lazy var cancellables: Set<AnyCancellable> = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        bind()
+        
+        // Do any additional setup after loading the view.
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: "Settings",
+            style: .plain,
+            target: self,
+            action: #selector(handleSettingsButton)
+        )
+
+        let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(insertNewObject(_:)))
+        navigationItem.rightBarButtonItem = addButton
+        if let split = splitViewController {
+            let controllers = split.viewControllers
+            detailViewController = (controllers[controllers.count - 1] as! UINavigationController)
+                .topViewController as? CombineSimpleChatViewController
+        }
+
+        tableView.addGestureRecognizer(longPressRecognizer)
+    }
+    
+    private func bind() {
```

Comment:
Please don't forget this is sample code for our customers. Can we use a more descriptive name for this? Like `setUpCombineSubscribers` or similar?